pywbem/_valuemapping.py

```diff
@@ -743,6 +796,11 @@ def tobinary(self, values_str):
         Return the integer value or values for a `Values` string, based upon
         this value mapping.
 
+        Due to the complexity of its return value, this method only supports a
+        single `Values` string at a time. It does support array-typed elements,
+        though. Thus, if multiple `Values` strings need to be translated, this
+        method simply needs to be invoked multiple times.
```

Comment:
Remove the word simply or better "method must be invoked once for each value to be translated"

Reply:
DONE